contracts/examples/esdt-transfer-with-fee/src/esdt_transfer_with_fee.rs

```diff
@@ -0,0 +1,122 @@
+#![no_std]
+
+mod fee;
+use fee::*;
+
+elrond_wasm::imports!();
+#[elrond_wasm::contract]
+pub trait EsdtTransferWithFee {
+    #[init]
+    fn init(&self) {}
+
+    #[only_owner]
+    #[endpoint(setExactValueFee)]
+    fn set_exact_value_fee(&self, fee: EsdtTokenPayment<Self::Api>, token: TokenIdentifier) {
```

Comment:
Better split the fee argument into token/value (they're all fungible, right? so no nonce), to make it easier to call the endpoint